src/mediatype-icon.ts

```diff
@@ -62,7 +70,7 @@ export class MediatypeIcon extends LitElement {
         text-align: center;
       }
 
-      svg {
+      #icon svg {
```

Comment:
What is the svg size behavior when it's not wrapped in the `icon` div?

Reply:
having it restricted by the size of containing `<div>` seems to work. TBD.